### PR TITLE
Add debit card acceptance signage to T&Cs

### PR DESCRIPTION
**Pull request by ushkarev**
Image for reference:

image

### CODE REVIEW

**Comment by mattbee** [comment]
Looks good to me. 

Thoughts that I dismissed: Responsive images could have been considered over loading both sets of images but given page this is (no one will go there!) and the size of the images being tiny and only a small number of browsers will load the images that are hidden, all is good.

File: mtp_send_money/templates/send_money/includes/card-acceptance-signage.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{% load static %}
+
+<div class="mtp-card-acceptance-signage__logo">
+  <svg width="{{ width }}" height="{{ height }}" role="presentation" focusable="false">
+    <image xlink:href="{% static 'images/card-acceptance-signage/visa.svg' %}" src="{% static 'images/card-acceptance-signage/visa@2x.png' %}" width="{{ width }}" height="{{ height }}" />
```
Comment (mattbee) [inline mtp_send_money/templates/send_money/includes/card-acceptance-signage.html]: I had reviewed but I had second thoughts about this - does `src` attribute do anything here (wasn't aware it was an attr for image tag), and `xlink:href` has been superseded by just `href` - is it worth changing that?

https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/xlink:href
https://developer.mozilla.org/en-US/docs/Web/SVG/Element/image

Comment (ushkarev) [reply mtp_send_money/templates/send_money/includes/card-acceptance-signage.html]: the src is used as a fallback for some older browsers. i'm using prior art from others [GDS' design system](https://github.com/alphagov/govuk-frontend/blob/master/src/govuk/components/header/template.njk) and [elsewhere](https://css-tricks.com/a-complete-guide-to-svg-fallbacks/). i agree, it's not official syntax, but allegedly works… i've tried all the devices available to me

Comment (ushkarev) [reply mtp_send_money/templates/send_money/includes/card-acceptance-signage.html]: i did initially try just a high-res image but the fine print was just too small

**Comment by ushkarev** [comment]
as for
> Responsive images could have been considered over loading both sets of images but given page this is (no one will go there!) and the size of the images being tiny and only a small number of browsers will load the images that are hidden, all is good.
the two responsive sets of images are actually the same static assets. the sizes apparently need to be set in the html not css (spotted in safari on ios) and cannot be percentages (allegedly, but i didn't confirm). but if you mean the fallback image, the link i referenced above says it depends on the browser what combo gets loaded, with newer ones only using svg. this _seems_ to be be most accepted/compatible pattern currently for svgs

File: mtp_send_money/templates/send_money/includes/card-acceptance-signage.html (hunk 1 above)
Comment (mattbee) [reply mtp_send_money/templates/send_money/includes/card-acceptance-signage.html]: 👍 Cool, makes sense then. Carry on :)

**Pull request merged**